virtualizarr/tests/test_parsers/test_hdf/test_hdf_integration.py

```diff
@@ -2,38 +2,37 @@
 import xarray as xr
 import xarray.testing as xrt
 
-import virtualizarr
-from virtualizarr.readers.hdf import HDFVirtualBackend
+from virtualizarr import open_virtual_dataset
+from virtualizarr.parsers import HDFParser
 from virtualizarr.tests import (
     requires_hdf5plugin,
     requires_icechunk,
     requires_imagecodecs,
     requires_kerchunk,
 )
-from virtualizarr.tests.test_integration import (
-    roundtrip_as_in_memory_icechunk,
-)
+from virtualizarr.tests.test_integration import roundtrip_as_in_memory_icechunk
+from virtualizarr.tests.utils import obstore_local
 
 
 @requires_kerchunk
 @requires_hdf5plugin
 @requires_imagecodecs
 class TestIntegration:
-    @pytest.mark.xfail(
-        reason="0 time start is being interpreted as fillvalue see issues/280"
-    )
     def test_filters_h5netcdf_roundtrip(
         self, tmp_path, filter_encoded_roundtrip_hdf5_file
     ):
+        store = obstore_local(file_url=filter_encoded_roundtrip_hdf5_file)
```

Comment:
@sharkinsspatial the use of prefixed stores via obstore_local is why the local files are now failing with my suggested changes because manifest store assumes no prefix in the chunk path parsing. Either we should disallow prefix stores in parses or add support for stripping the store prefix from the chunk key path in ManifestStore.